Example/HCaptcha_Tests/Core/HCaptcha__Bench.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  HCaptcha__Bench.swift
+//  HCaptcha_Tests
+//
+//  Copyright © 2022 HCaptcha. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import HCaptcha
+
+class HCaptcha__Bench: XCTestCase {
+    override func setUp() {
+        HCaptchaHtml.template = """
+            <html>
+              <head>
+                <meta name="viewport" content="width=device-width" />
+                <script type="text/javascript">
+                  var apiKey = "${apiKey}";
+                  var endpoint = "${endpoint}";
+                  var rqdata = "${rqdata}";
+                  var theme = ${theme};
+                  var debugInfo = JSON.parse('${debugInfo}');
+
+                  console.assert(typeof apiKey === "string", "invalid apiKey");
+                  console.assert(typeof endpoint === "string", "invalid endpoint");
+                  console.assert(typeof rqdata === "string", "invalid rqdata");
+                  console.assert(["string", "object"].includes(typeof theme), "invalid theme object");
+                  console.assert(Array.isArray(debugInfo), "invalid type");
+
+                  var post = function(value) {
+                    window.webkit.messageHandlers.hcaptcha.postMessage(value);
+                  };
+
+                  var execute = function() {
+                    post({ token: "bench-token" });
+                  };
+
+                  var reset = function() {
+                    post({ action: "didLoad" });
+                  };
+
+                  post({ action: "didLoad" });
+                </script>
+              </head>
+              <body>
+                <div id="hcaptcha-container">
+              </body>
+            </html>
+        """
+    }
+
+    let options: XCTMeasureOptions = {
+        var result = XCTMeasureOptions()
+        result.iterationCount = 100
```

Comment:
Probably excessive. I'd go with 3 if we're running this in CI.

Reply:
This is exactly why I set it more than 10 (default value), because a couple of first runs especially in CI can be extremely slow

Reply:
I think it would bloat the CI time to run 100x, so probably best to leave a manual option.

Also, this is doing an e2e benchmark (including net latency etc) but what we mainly care about measuring is the SDK itself. In other words, time from setup to webview init.

You can capture this by injecting a line of JS in the body that calls the bench-token, rather than putting it in execute.

Reply:
> I think it would bloat the CI time to run 100x, so probably best to leave a manual option.

hmm, I disagree but ok, I will set it to 3

> Also, this is doing an e2e benchmark (including net latency etc) but what we mainly care about measuring is the SDK itself. In other words, time from setup to webview init.

Right I think that SDK repo probably isn't the best place to do such kind of testing (I mean latency) it should be done somewhere else i.e. js-sdk repo or similar (to be able to check latency from different locations also)

> You can capture this by injecting a line of JS in the body that calls the bench-token, rather than putting it in execute.

In this case, a token will be received right after init (no validation call will be required) is this really what we want?

Reply:
Yes: the goal is to benchmark the iOS SDK by itself, excluding the JS code: that has its own separate benchmarks.

Reply:
I totally agree with this, and I have tried to keep JS as minimal as possible, In my understanding, if we will remove `execute` from js we will not do testing properly

Reply:
An e2e benchmark could also be useful, but it's different than the goal of this issue.

What we specifically want to find are perf regressions in the iOS SDK itself. 

Including the JS code will also introduce a lot of jitter in measurements since it involves live network calls etc. That's out of scope here.

Reply:
Currently for benchmarking we use this template https://github.com/hCaptcha/HCaptcha-ios-sdk/pull/95/files#diff-90daa2012f960ed6b7a77ece4870a58c8e461cc131af84897987227b048f8b15R15-R49 and it doesn't lead to any network calls